services/core/shared/core/transactionStatistics.js

```diff
@@ -142,8 +143,8 @@ const fetchTransactions = async (date, offset = 0) => {
 		limit,
 		offset,
 	};
-	const transactions = await getTransactions(params);
-	return transactions.data;
+	const transactions = await requestAll(getTransactions, params, 20000);
```

Comment:
Instead of hardcoding the limit value at 20000, it might be better to invoke `getTransactions` with `params` (but with `limit: 1`, eg: `{ ...params, limit: 1 }`) and pass `meta.total` from the response to `requestAll` as the limit.

Reply:
Done!